### ISSUE TITLE
Context menu not visible

### ISSUE THREAD

**Issue by eugenesvk** [Predelnik/DSpellCheck#202]
I'm trying to understand how to use a convenient context menu to spellcheck/switch languages
> Ability to quickly change current language through context menu or DSpellCheck sub-menu

But when I click on a misspelled word, I don't see any context menu items from this addon. Here is a screenshot of all the context menu items I have
![ContextMenu](https://user-images.githubusercontent.com/12956286/74235637-8d95a000-4ce0-11ea-89e0-27d09490d161.png)

How do I enable **DSpellCheck** context menu?
(I'm able to use the regular Menu, also I'm able to use the **Special Suggestion Button**

<details>
<summary>System Info</summary>

Notepad++ v7.8.4   (64-bit)
Build time : Jan 29 2020 - 01:31:03
Admin mode : OFF
Local Conf mode : ON
OS Name : Windows 10 Pro (64-bit), 1909, 18363.592
Plugins : CodeAlignmentNpp.dll DSpellCheck.dll ElasticTabstops.dll ExtSettings.dll Linter.dll MarkdownViewerPlusPlus.dll MenuIcons.dll mimeTools.dll NavigateTo.dll nppAutoDetectIndent.dll NppConverter.dll NppExport.dll NppMenuSearch.dll NppSaveAsAdmin.dll PythonScript.dll XBrackets.dll zoomdisabler_x64.dll _CustomizeToolbar.dll 

</details>

**Comment by Predelnik** [comment]
Probably it's some problem with interaction between plugins, I need to check which one from the ones you are using breaks this. Unfortunately integrating into context menu is not exactly "officially" supported through plugin system so some problems like this might happen.

**Comment by eugenesvk** [comment]
Hm, you're right, I've just tested with an empty version and it works. By the way, I've noticed there is also a contextMenu.xml file, is that something that could be used to manually plug your plugin's token ids or something?
I've done some quick testing and found the offending plugin: [MenuIcons](https://sites.google.com/site/fstellari/nppplugins
)

**Comment by Predelnik** [comment]
I'm not sure that I can rewrite `contextMenu.xml` it looks like it was made for user customization for the most part and anyway for spell-check there are some rather special requirements for spell-checking menu since it is dynamic etc., I could probably settle for modifying one sub-menu but it still might not be as convenient.

Thank you for finding the problematic plugin, I will do my best to make my interaction with context menu work when `MenuIcons` is present and will try to request the changes if they are needed on `MenuIcons`'s authors side.

**Comment by eugenesvk** [comment]
Yeah, I totally get the challenge of trying to fit a rather useful dynamic menu into static constraints, and sub-menu is indeed not as convenient, I wrote this before I found the offending plugin, for now I've just removed the MenuIcons plugin as it's not as important as yours are.
Hopefully you'll find a way to make it work! Thanks

**Comment by Predelnik** [comment]
After checking this, it seems to be a problem on `MenuIcons`'s side. Since `MenuIcons` is closed source I sent an e-mail to the author with suggestion on how it most likely needs to be fixed.

**Comment by SN-CH** [comment]
It seems this Menu icons plug-in (version 1.22) is even more broken than it has been guessed. I can't even uninstall it with the internal Plug-ins admin (I use the latest N++ 7.9.5 x32).

However, it is maybe not even necessary. In order to see the spell checking items in the context menu one should go to Menu icons settings, select the «Flat (same folder)» radio-button
![Screen-shot_2021-05-01_18-47-08](https://user-images.githubusercontent.com/42577865/116787623-b654b180-aaad-11eb-9dc3-c2698c60c212.png)
 and re-launch N++. Regrettably, one won't get icons in the menus after that, but he is rewarded with having spell checking items back in the context menu in stead.

It is sort of weird, but has worked for me.
